server/src/test/java/org/opensearch/search/query/QueryProfilePhaseTests.java

```diff
@@ -336,12 +336,12 @@ public void testMinScoreDisablesCountOptimization() throws Exception {
         assertEquals(TotalHits.Relation.EQUAL_TO, context.queryResult().topDocs().topDocs.totalHits.relation);
         assertProfileData(context, "MatchAllDocsQuery", query -> {
             assertThat(query.getTimeBreakdown().keySet(), not(empty()));
-            assertThat(query.getTimeBreakdown().get("score"), greaterThanOrEqualTo(100L));
+            assertThat(query.getTimeBreakdown().get("score"), greaterThanOrEqualTo(1L));
             assertThat(query.getTimeBreakdown().get("score_count"), equalTo(1L));
             if (executor != null) {
-                assertThat(query.getTimeBreakdown().get("max_score"), greaterThanOrEqualTo(100L));
-                assertThat(query.getTimeBreakdown().get("min_score"), greaterThanOrEqualTo(100L));
-                assertThat(query.getTimeBreakdown().get("avg_score"), greaterThanOrEqualTo(100L));
+                assertThat(query.getTimeBreakdown().get("max_score"), greaterThanOrEqualTo(1L));
+                assertThat(query.getTimeBreakdown().get("min_score"), greaterThanOrEqualTo(1L));
+                assertThat(query.getTimeBreakdown().get("avg_score"), greaterThanOrEqualTo(1L));
```

Comment:
Maybe we should check `min <= avg <= max` instead?

Reply:
Only checking `min <= avg <= max` is insufficient here. We must ensure that they have positive values.